src/game.cpp

```diff
@@ -2726,20 +2532,17 @@ void Game::toggleFreeMove()
 	bool free_move = !g_settings->getBool("free_move");
 	g_settings->set("free_move", bool_to_cstr(free_move));
 
-	runData.statustext_time = 0;
-
 	if (free_move) {
 		if (client->checkPrivilege("fly")) {
-			showStatusTextSimple("Fly mode enabled");
+			m_game_ui->showTranslatedStatusText("Fly mode enabled");
```

Comment:
Needs to be added to our translation updating script, so these texts are detected.

Reply:
done